Tasks/Kuchinskiy/Task9/oop_logic.py

```diff
@@ -0,0 +1,81 @@
+import json
+# в этом файле треним разные способы написания классов, 
+# поэтому функционал программы 
+# может осуществляться разными способами
+# 1) поиск фильмов и возврат всех фильмов осуществляется через classmethod
+# 2) запись: используем объект класса и добавляем в файл
+# 3) удаление: создание объектов класса используя данные из файла
+class Movie:
+    try: # создаем атрибут класса, чтобы работать с ним в рамках класса
+        with open('movies.json', 'r') as f:
+            new_file = json.load(f)
+    except(FileNotFoundError, json.decoder.JSONDecodeError):
+        new_file = []
+
+    @classmethod # поиск фильмов
+    def search_movie(cls, name, keys):
+        output = []
+        for dic in cls.new_file:
+            if dic[keys] == name:
+                output.append([val for val in dic.values()])
+        return output
+
+    @classmethod # вернуть все фильмы
+    def get_all_movies(cls):
+        output = []
+        for dic in cls.new_file:
+            output.append([val for val in dic.values()])
+        return output
+            
+
+    def __init__(self, title, director, year, genre):
+        self.__title = title
+        self.__director = director
+        self.__year = year
+        self.__genre = genre
```

Comment:
this is a bad approach, your class has too many responsibilities it's a container of a singular movie data but it's also a 'manager' for the storage task, it would be MUCH better to have separate data types for the separate tasks